inference-engine/src/low_precision_transformations/src/common/mat_mul.cpp

```diff
@@ -95,7 +95,7 @@ bool MatMulTransformation::transform(TransformationContext &context, ngraph::pat
                 NetworkHelper::toScalar(as_type_ptr<opset1::Constant>(const1)),
                 const2)));
     replace_node(matMul, newMultiply);
-
+    newMultiply->set_friendly_name(matMul->get_friendly_name() + "/DequantizationMultiply");
```

Comment:
Could you please explain the logic in which cases we need to update the name.  Why don't you do the same inside `add` transformation for example?

Reply:
was removed, not the goal of this commit